### PR TITLE
feat(translations): Added Armenian language

### PR DESCRIPTION
**Pull request by shahen94**
*Summary*

Added new translation metadata - _Armenian_

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @shahen94, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#3616).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by shahen94** [comment]
I got an infinite loop

image

**Comment by shahen94** [comment]
Screen Shot 2019-09-04 at 7 17 40 PM

**Comment by bramkragten** [comment]
Looks like `unpkg.com` gave an error, can you load https://unpkg.com/github-api@3.3.0/dist/GitHub.bundle.min.js ?

**Comment by shahen94** [comment]
@bramkragten  Yes, the file is accessible

**Comment by shahen94** [comment]
@bramkragten I tried to sign up through the link again and it worked!
Thanks

**Comment by shahen94** [comment]
PS. I'm not sure what should be my next steps to add language support.
I checked `lokalise` dashboard and couldn't find a way to create a new language there.

Can someone please give me some clues?

Thanks in advance.

File: src/translations/translationMetadata.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "af": {
     "nativeName": "Afrikaans"
   },
+  "am": {
```
Comment (balloob) [inline src/translations/translationMetadata.json]: The language code is incorrect. It should be hy.

image

Comment (shahen94) [reply src/translations/translationMetadata.json]: Fixed, thanks!

**Review by balloob — APPROVED** [review]

**Comment by balloob** [comment]
Language added to Lokalise too. Happy translating!

**Comment by shahen94** [comment]
@balloob Thank you!
Are you sure that you added language in Lokalize too? I couldn't find it in the list.
image

**Comment by balloob** [comment]
image

**Comment by balloob** [comment]
ugh, looks like I need to give existing users access to it. Gave you access.

**Pull request merged**